MonoGame.Framework/MonoGame.Framework.DesktopGL.csproj

```diff
@@ -0,0 +1,218 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <TargetFrameworks>netstandard2.0;net451</TargetFrameworks>
+    <RootNamespace>Microsoft.Xna.Framework</RootNamespace>
+    <AssemblyName>MonoGame.Framework</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PackageIconUrl>https://pbs.twimg.com/profile_images/487954549441691649/O3KsHAsb_400x400.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/cra0zy/MonoGame/tree/core</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/cra0zy/MonoGame/tree/core</RepositoryUrl>
+    <Description>.NET Standard version of MonoGame DesktopGL platform.</Description>
+    <PackageTags>monogame;.net core;core;.net standard;standard;desktopgl</PackageTags>
+    <Version>1.0.0.0</Version>
+    <Authors>cra0zy</Authors>
+    <PackageId>MonoGame.Framework.DesktopGL</PackageId>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <OutputPath>bin\DesktopGL\AnyCPU\Debug</OutputPath>
+    <IntermediateOutputPath>obj\Linux\AnyCPU\Debug</IntermediateOutputPath>
+    <DocumentationFile>bin\DesktopGL\AnyCPU\Debug\MonoGame.Framework.xml</DocumentationFile>
+    <DefineConstants>DEBUG;OPENGL;OPENAL;XNADESIGNPROVIDED;TRACE;LINUX;DESKTOPGL;SUPPORTS_EFX;NETSTANDARD</DefineConstants>
```

Comment:
DEBUG is added automatically if the configuration is Debug.

We don't need two property groups here. Only difference is DocumentationFile and that can be set with `$(Configuration)` to differentiate between Debug and Release